iphone/Classes/TiUIListView.m

```diff
@@ -1102,12 +1102,9 @@ - (BOOL)canMoveRowAtIndexPath:(NSIndexPath *)indexPath
   UIDragItem *dragItem = [[UIDragItem alloc] initWithItemProvider:itemProvider];
   dragItem.localObject = identifier;
 
-  return @[ dragItem ];
-}
-
-- (void)tableView:(UITableView *)tableView dropSessionDidEnter:(id<UIDropSession>)session
-{
   [[self proxy] fireEvent:@"movestart"];
+
+  return @[ dragItem ];
```

Comment:
Just wondering, did you give [`dragSessionWillBegin`](https://developer.apple.com/documentation/uikit/uitableviewdragdelegate/2897382-tableview) a try? I think technically it would be more correct to put it there because then it's clear a drag is about to start. Or is that too late as well?

Reply:
`dragSessionWillBegin` is only called once the actual drag starts to move from it's initial index position (which is much later than we know about its movement now). Regarding `canMoveRowAtIndexPath`: I don't see any relation to the `itemsForBeginningDragSession` delegate in which it is called right now. Can you elaborate on that?

Reply:
@janvennemann We ok then? Happy to move it to a better place if availble

Reply:
@hansemannn My bad, the GitHub diff had me confused because it showed the wrong method name. All good then!